### PR TITLE
403 patient tabs

### LINKED ISSUE #403: Patient tab interface
Add a patient tab interface and rewired the programs tabs to the current programs list view.

Image

### PR DESCRIPTION
**Pull request by mark-prins**
Closes #403 

Adds in the tabs for the patient view.
and.. while I'm at it:
- patient summary block
- breadcrumb shows the patient name

I also removed the H1 in my copy of the patients-ui-schema to make it look like this:

Screen Shot 2022-08-02 at 4 32 59 PM

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/patients.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
   "button.new-patient": "New Patient",
   "filename.patients": "patients",
+  "label.date-of-birth": "DOB (Age)",
+  "label.details": "Details",
+  "label.gender": "Gender",
```
Comment (clemens-msupply) [inline client/packages/common/src/intl/locales/en/patients.json:6]: This is requested to be "Sex". Do we usually call it gender? do we need to make it configurable?

**Review by clemens-msupply — APPROVED** [review]
Looks good thank you!

What do you think about the reduced space for the detail view? the app bar takes quite a bit of space and is mostly empty (don't have a better idea though).

File: client/packages/common/src/intl/locales/en/patients.json (hunk 1 above)
Comment (mark-prins) [reply client/packages/common/src/intl/locales/en/patients.json:6]: we've used `Gender` previously and to me it's a gentler term to use for this. Given that these changes are specific for a client, they get to call the shots though.. I guess.

**Comment by mark-prins** [comment]
> What do you think about the reduced space for the detail view?

Yeah, it's not great. We could 
- remove the patient summary
- squash it all on to one line
- fit the tabs in the same area
- reduce the summary font size

For me, the current layout looks the best, at the sacrifice of some screen real estate. and the summary is useful when on another tab. It's redundant on the detail tab though

**Pull request merged**